subprojects/build-init/src/testFixtures/groovy/org/gradle/buildinit/plugins/fixtures/ScriptDslFixture.groovy

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradle.buildinit.plugins.fixtures
+
+import org.gradle.api.JavaVersion
+import org.gradle.buildinit.plugins.internal.modifiers.BuildInitBuildScriptDsl
+import org.gradle.test.fixtures.file.TestFile
+import org.gradle.util.GradleVersion
+import org.hamcrest.Matcher
+
+import static org.gradle.buildinit.plugins.internal.modifiers.BuildInitBuildScriptDsl.GROOVY
+import static org.gradle.buildinit.plugins.internal.modifiers.BuildInitBuildScriptDsl.KOTLIN
+import static org.hamcrest.Matchers.containsString
+
+class ScriptDslFixture {
```

Comment:
I may be totally wrong, but this looks like the entire file is using static methods? Consider annotating with `@CompileStatic` to speed up the invocation of all of these methods?

Reply:
Good call. This is not production code but we love faster tests, don't we?
Done ✅

Reply:
I see your "Done" but I don't see the annotation added?